packages/component-library-react/src/Calendar/Calendar.test.tsx

```diff
@@ -0,0 +1,153 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { formatISO } from 'date-fns';
+import { nl } from 'date-fns/locale';
+import { Calendar, Events } from './index';
+
+describe('Calendar', () => {
+  it('invokes an onclick callback with a selected date', () => {
+    let selectedDate = '';
+    render(
+      <Calendar
+        currentDate={new Date(2023, 5, 11)}
+        onCalendarClick={(date) => {
+          selectedDate = date;
+        }}
+        locale={nl}
+      />,
+    );
+
+    let dayButton = screen.getByText('15');
+    fireEvent.click(dayButton);
+
+    expect(selectedDate).toEqual(formatISO(new Date(2023, 5, 15)));
+  });
+
+  it('renders a design system BEM class name', () => {
+    const { container } = render(<Calendar onCalendarClick={() => {}} />);
+
+    const calendar = container.querySelector(':only-child');
+
+    expect(calendar).toHaveClass('utrecht-calendar');
+  });
+
+  it('renders the current date', () => {
+    render(<Calendar onCalendarClick={() => {}} locale={nl} />);
```

Comment:
Hmm if we render without setting currentDate I don't think the hard-coded aria-label will work, but it's probably fine to add currentDate here?